lib/prerequisites.sh

```diff
@@ -92,10 +94,10 @@ if [ "${PROJECT}" != "minimal" ]; then
   fi
 fi
 checkPrivKey() {
-  if [ -e /etc/bridgehead/pki/${SITE_ID}.priv.pem ]; then
+  if [ -e "${CONFIG_DIR}"pki/"${SITE_ID}".priv.pem ]; then
```

Comment:
This should be like so no?
```suggestion
  if [ -e "${CONFIG_DIR}pki/${SITE_ID}.priv.pem" ]; then
```

Reply:
It should expand to the same thing, but is the cleaner solution. I agree.